ingest/src/main/java/com/microsoft/azure/kusto/ingest/Validation.java

```diff
@@ -0,0 +1,48 @@
+package com.microsoft.azure.kusto.ingest;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public class Validation {
+
+    public static void validateIsNotBlank(String str, String message) {
```

Comment:
IsNotNullOrWhiteSpace? Like in .Net

Reply:
The isBlank() in Java checks if it is null, empty, or white space.
Anyway, I changed the name of it (as per your request in another comment) to stringIsNotBlank(), to align with other renamed functions like: argIsNotNull() and fileExists().